clients/erigon/Caddyfile.template

```diff
@@ -0,0 +1,41 @@
+{
+	admin 127.0.0.1:2019 {
+	}
+	log {
+		output stdout
+		format json
+		level INFO
+	}
+	acme_dns cloudflare {$CLOUDFLARE_API_KEY}
+	auto_https disable_redirects
+	storage file_system {{ data_dir }}
+	email ops@blockjoy.com
+	servers {
+		metrics
+	}
+}
+
+http://{{ hostname }}{{ tld }}, https://{{ hostname }}{{ tld }}, *.blkjy.io, http:// {
+	log {
```

Comment:
```suggestion
	tls {
		dns cloudflare {$CLOUDFLARE_API_KEY}
		resolvers 1.1.1.1
	}
	log {
```

Reply:
We should include this, this will prevent the nodes from trying to use 127.0.0.1 for dns lookups, which doesn't work and helps gets certs.

Reply:
Also there's a typo in the variable syntax